studio/src/app/handlers/editor/events/deck/deck-events.handler.tsx

```diff
@@ -542,6 +542,13 @@ export class DeckEventsHandler {
             if ((slide as any).imgAlt) {
                 attributes.imgAlt = (slide as any).imgAlt;
             }
+            
+            if (slide.getAttribute('vertical') === 'true') {
```

Comment:
I would rather like that you test here the type of node and also, if not provided, then clean it from the db (`firebase.firestore.FieldValue.delete()`).

You could have a look to method `getSlideAttributesQRCode` and adapt for `split`

Reply:
How about this?

```tsx
if ((slide as any).vertical) {
    attributes.vertical = true;
} else if (cleanFields) {
    // @ts-ignore
    attributes.vertical = firebase.firestore.FieldValue.delete();
}
```

Reply:
Sure it works too